synchronizer/batches.go

```diff
@@ -326,21 +324,23 @@ func (bs *BatchSynchronizer) resolve(batch types.BatchKey) (*types.OffChainData,
 	}
 
 	// pull out the members, iterating will change the map on error
-	members := make([]etherman.DataCommitteeMember, len(bs.committee))
-	for _, member := range bs.committee {
+	members := make([]etherman.DataCommitteeMember, bs.committee.Length())
+	bs.committee.Range(func(_ common.Address, member etherman.DataCommitteeMember) bool {
```

Comment:
What are the objectives of this kind of implementation rather than just "Copy()" function that returns the copied slice of members?

Reply:
That's another approach, but the mindset behind the decision was that having the `Range` function in place, is a more generic approach, as it can be reused, than having the `Copy` function.

The main idea behind this PR is to provide a thread-safe map for the committee members (namely reading and altering the map should be thread-safe). Does it make sense?

Reply:
The scope of DA is clearly defined and relatively limited. Do we have any other places in the code where we are going (potentially) to use this function? 

Currently, it looks a little bit overkill IMO.

Reply:
it's an inline func, how to reuse? I vote for the Copy() as @begmaroman mention

Reply:
> Do we have any other places in the code where we are going (potentially) to use this function?
Currently probably not. Just to make this clear: do you mind only not providing `Copy` function, or do you feel that we do not need the PR as a whole?

> it's an inline func, how to reuse?
The `Range` function can be reused (as it is a thread-safe iterator with a custom callback).

But I can provide the `Copy` function if you prefer it better and remove the `Range` function altogether. :slightly_smiling_face:

Reply:
Fixed https://github.com/0xPolygon/cdk-data-availability/pull/96/commits/b9fd7aa1ee683f551088f29b284120742f94f4b4